src/components/IOUConfirmationList.js

```diff
@@ -285,11 +277,15 @@ class IOUConfirmationList extends Component {
         const shouldDisableButton = selectedParticipants.length === 0;
         const recipient = this.state.participants[0];
         const canModifyParticipants = this.props.canModifyParticipants && this.props.hasMultipleParticipants;
+        const formattedAmount = this.props.numberFormat(this.props.iouAmount, {
+            style: 'currency',
+            currency: this.props.iou.selectedCurrencyCode,
+        });
 
         return (
             <OptionsSelector
                 sections={this.getSections()}
-                value={this.props.comment}
+                value=""
```

Comment:
It looks a bit odd to pass this static value. Can we remove this prop since we are not rendering the `textInput` in this case?

Reply:
It's a required prop though. Not sure if I wanted to mess with that or not 🤔

Reply:
Seems fine to leave it as is then. Thanks for considering this :)